joint_limits:
  j1:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 0.5
  j2:
    has_position_limits: true
    min_position: -2.20
    max_position: 0.40
    has_velocity_limits: true
    max_velocity: 0.5
  j3:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 0.5
  j4:
    has_position_limits: true
    min_position: -1.57
    max_position: 1.57
    has_velocity_limits: true
    max_velocity: 0.5